.github/workflows/build-tool.yml

```diff
@@ -6,50 +6,98 @@ on:
       dotnet_sdk_version:
         required: true
         type: string
-      REPOSITORY_NAME:
-        required: true
-        type: string
-      MORYX_OPTIMIZE_CODE:
-        required: false
-        type: string
       MORYX_BUILD_CONFIG:
         required: false
         type: string
-      MORYX_BUILDNUMBER:
-        required: false
+      REPOSITORY_NAME:
+        required: true
         type: string
-      DOTNET_SKIP_FIRST_TIME_EXPERIENCE:
-        required: false
-        type: boolean
-
+ 
 jobs:
   Build:
-    runs-on: windows-latest
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
       - name: Setup .NET SDK
```

Comment:
I guess this step still needs to be removed?

Reply:
Which part? The build-part definitely needs to be included :)

Reply:
No I meant that similar to the test step here we setup the .net 6 and 7 sdks. I mean its fine if both only work with .net 6 but then you could just remove the .net 7 setup step, couldn't you? 😅